README.md

```diff
@@ -72,19 +67,10 @@ vars:
 
 For additional configurations for the source models, such as the company and tag variables, visit the [Intercom source package](https://github.com/fivetran/dbt_intercom_source).
 
-## Contributions and Future Considerations
-Currently Intercom V2.0 does not support API exposure to company defined business hours. As such, 
-all time_to metrics are calculated in their entirety without subtracting out business hours. A feature 
-ticket has been logged with Intercom to provide API exposure for business hours in a later release.
-Don't see a model or specific metric you would have liked to be included? Notice any bugs when installing 
-and running the package? If so, we highly encourage and welcome contributions to this package! 
-Please create issues or open PRs against `master`. Check out [this post](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) 
+## Contributions
```

Comment:
@fivetran-joemarkiewicz We do not publish future considerations - we just advertise updates when they happen.

Reply:
Although we do not publish future considerations, would we be able to keep the verbiage regarding the Intercom V2.0 API not supporting exposure to company defined business hours? I feel this will be a frequent question and would be good to have a disclaimer that time metrics do not take into account business hours. 

Possibly adding to the "NOTE:" at the beginning of the README?

Reply:
@fivetran-chloe reporting on business hours came up several times during our customer calls so I think it's important to include this info.  Perhaps we can phrase it as a "limitation" of the package rather than a "future consideration"?

Reply:
@fivetran-kristin @fivetran-joemarkiewicz Got it. I'm hesitant to add any more to the note at the top of the page, so I think a "Limitations" section is the way to go. Can you please add it?

Reply:
@fivetran-kristin @fivetran-chloe I added a "Limitations" section to the README. I made sure to not mention any future considerations for adding business hours when it does get API exposure. Let me know if you feel the Limitations section should be updated.

Reply:
@fivetran-joemarkiewicz Thanks for adding the Limitations section! I've made a few minor edits for clarity. If my edits look good to you, then this PR is ready to merge.